ext/drcovlib/modules.c

```diff
@@ -234,7 +236,7 @@ static inline void
 lookup_helper_set_fields(module_entry_t *entry, OUT uint *mod_index, OUT app_pc *mod_base)
 {
     if (mod_index != NULL)
-        *mod_index = entry->id;
+        *mod_index = entry->containing_id; /* Yes, the main (containing) module. */
```

Comment:
removed.
I want to return the consistent result.
Currently, it could return either the segment or the containing module.